test/demo/index.tsx

```diff
@@ -130,6 +144,41 @@ const Page: FC<{
         </ul>
       </>
     )
+  } else if (router === 'list 1') {
```

Comment:
Let’s use `list1`. I prefer to avoid spaces in IDs.

Reply:
done